packages/tupaia-web/src/pages/Login.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import { useLogin } from '../api/mutations';
+import { TextField } from '../components/TextField';
+import { AuthModal, ModalButton } from '../layout';
+
+const StyledForm = styled.form`
+  margin-top: 1rem;
+  width: 340px;
+  max-width: 100%;
+`;
+
+const LinkText = styled(Typography)`
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 15px;
+  color: white;
+
+  a {
+    color: white;
+  }
+
+  ${ModalButton} + & {
+    margin-top: 1.3rem;
+  }
+`;
+
+const ForgotPasswordText = styled(LinkText)`
+  display: block;
+  margin-top: -0.6rem;
+  text-align: right;
+`;
+
+export const Login = () => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm();
+  const { mutate: login, isLoading, isError, error } = useLogin();
+  const navigate = useNavigate();
+
+  const onClose = () => {
+    navigate(-1);
```

Comment:
I wonder if we need to do a check here as well, to make sure navigating doesn't take the user back to another site or an empty tab, if they go directly to `/login`?